addons/udes_delivery_control/models/picking_extra_fields.py

```diff
@@ -114,6 +119,22 @@ def _compute_backload_time_taken(self):
                     record.backload_start_date, record.backload_end_date
                 )
 
+    @api.depends("loading_type")
+    # @api.onchange("u_loading_type")
```

Comment:
Nit: Commentated out `onchange` can be removed.

Reply:
done!